1.8/administration/access-node/tunnel.md

```diff
@@ -0,0 +1,113 @@
+---
+post_title: Access by Proxy
+menu_order: 10
+---
+
+When developing applications on DC/OS, you may find it helpful to access your cluster via SOCKS proxy, HTTP proxy, or VPN. For instance, you can work from your own development environment and immediately test on your DC/OS cluster.
```

Comment:
I think this might make more sense as "test against" rather than "test on". "test on" makes it sound like the proxy will help you run stuff on DC/OS rather than "testing against" a service that is already running on DC/OS (using local debugging tools from your development environment)